services/notifyOthers.ts

```diff
@@ -1,16 +1,19 @@
 import { AccountInfo } from "@azure/msal-browser";
-import { getUserShortName } from "../utils/getUserShortName";
-import { getAccessToken } from "../auth/msalHelpers";
+import { getUserShortName } from "@/utils/getUserShortName";
+import { getAccessToken } from "@/auth/msalHelpers";
 
 export const notifyOthers = async (
   msg: unknown,
   roomId: string | string[],
   account?: AccountInfo | null
 ) => {
   // build message obj
+  let user = "";
+  if (account) user = getUserShortName(account);
   const message = {
     msg,
     roomId,
+    user,
   };
 
   if (account) message["user"] = getUserShortName(account);
```

Comment:
I think you should be able to do this: 
user: account ? getUserShortName(account) : "",

Then you can delete the "if" statements

Reply:
fixed